src/Application/Products/Commands/AddProduct/AddProductCommandHandler.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using MediatR;
```

Comment:
I guess we could make this a `global using MediatR` in a `GlobalUsings.cs`, because it's used in most files in this project.

Reply:
Done